18-minimal/Dockerfile.fedora

```diff
@@ -0,0 +1,66 @@
+FROM quay.io/fedora/fedora-minimal:38
+
+EXPOSE 8080
+
+# Add $HOME/node_modules/.bin to the $PATH, allowing user to make npm scripts
+# available on the CLI without using npm's --global installation mode
+# This image will be initialized with "npm run $NPM_RUN"
+# See https://docs.npmjs.com/misc/scripts, and your repo's package.json
+# file for possible values of NPM_RUN
+# Description
+# Environment:
+# * $NPM_RUN - Select an alternate / custom runtime mode, defined in your package.json files' scripts section (default: npm run "start").
+# Expose ports:
+# * 8080 - Unprivileged port used by nodejs application
+ENV APP_ROOT=/opt/app-root \
+    # The $HOME is not set by default, but some applications need this variable
+    HOME=/opt/app-root/src \
+    NPM_RUN=start \
+    PLATFORM="fedora" \
+    NODEJS_VERSION=18 \
+    NPM_RUN=start \
+    NAME=nodejs
+
+ENV SUMMARY="Minimal image for running Node.js $NODEJS_VERSION applications" \
+    DESCRIPTION="Node.js $NODEJS_VERSION available as container is a base platform for \
+running various Node.js $NODEJS_VERSION applications and frameworks. \
+Node.js is a platform built on Chrome's JavaScript runtime for easily building \
+fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model \
+that makes it lightweight and efficient, perfect for data-intensive real-time applications \
+that run across distributed devices." \
+    NPM_CONFIG_PREFIX=$HOME/.npm-global \
+    PATH=$HOME/node_modules/.bin/:$HOME/.npm-global/bin/:$PATH
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Node.js $NODEJS_VERSION Micro" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,$NAME,${NAME}${NODEJS_VERSION}" \
+      io.openshift.s2i.scripts-url="image:///usr/libexec/s2i" \
+      io.s2i.scripts-url="image:///usr/libexec/s2i" \
+      com.redhat.dev-mode="DEV_MODE:false" \
+      com.redhat.deployments-dir="${APP_ROOT}/src" \
+      com.redhat.dev-mode.port="DEBUG_PORT:5858" \
+      com.redhat.component="${NAME}-${NODEJS_VERSION}-minimal-container" \
+      name="fedora/$NAME-$NODEJS_VERSION-minimal" \
+      version="1" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
+      help="For more information visit https://github.com/sclorg/s2i-nodejs-container"
+
+# nodejs-full-i18n is included for error strings
+RUN INSTALL_PKGS="nodejs nodejs-nodemon nodejs-full-i18n nodejs-npm findutils tar nss_wrapper-libs" && \
```

Comment:
`which` is missing here. This package/command is added to the other Dockerfiles. Is it intentional?